### DISCUSSION TITLE
Notification message not visible when i send image url

### DISCUSSION THREAD

**Discussion by debadaracco** [invertase/react-native-firebase#4376] [Q&A]
When i send a notification push to my app by fcm and the notification data is received OK (Title, message and image url). The collapsed notification shows all information ok but when system shows the expanded notification only the title and image are visible while message disappears.

Device:
Moto E2 - Android 6.0

![Push Notification Collapsed](https://imagizer.imageshack.com/img924/3834/swR2IR.png)

![Push Notification Expanded](https://imagizer.imageshack.com/img922/2348/eLm5Fv.png)

Somebody knows how to solve this problem?

Thanks

**Comment by mikehardy** [comment]
Hi there!
So if I'm your worker, and I come to you with this problem, you have some questions right?
What versions of everything are you using? Maybe you need to update to current stable?
What's the *exact* JSON payload sent via the command line to the FCM REST API endpoint so you can reproduce this?
Since it's a notification payload (I think? You don't include the payload I'm just guessing) and we know that notification payloads are handled completely by the underlying firebase-android-sdk, do you see the same when you implement it without react-native-firebase, via a quickstart for the SDK? https://github.com/firebase/quickstart-android/tree/master/messaging

Since device-specific notifications is not something firebase actually handles (it's a non-firebase API surface area, really), does this happen when you integrate a full-featured 3rd party notification package (like react-native-push-notifications or notifee)?

I personally use data-only JSON payloads and a 3rd party notification package normally, and don't have issues
